networkit/stopwatch.py

```diff
@@ -8,7 +8,8 @@
 
 import time
 
-"""stopwatch is a very simple Python module for measuring time.
```

Comment:
Same here, do we need NetworKit to implement a Python stopwatch? I think this is another feature we should deprecate.